modules/layers/src/base-tile-layer/base-tile-layer.js

```diff
@@ -0,0 +1,156 @@
+import {log} from '@deck.gl/core';
+import {CompositeLayer} from '@deck.gl/core';
+import GeoJsonLayer from '../geojson-layer/geojson-layer';
+import TileCache from './utils/tile-cache';
+
+const defaultProps = {
+  renderSubLayers: {type: 'function', value: props => new GeoJsonLayer(props), compare: false},
+  getTileData: {type: 'function', value: ({x, y, z}) => Promise.resolve(null), compare: false},
+  tile2boundingBox: {
+    type: 'function',
+    value: ({x, y, z}) => {
+      return {east: 0, west: 0, north: 0, south: 0};
+    }
+  },
```

Comment:
- I'd leave the default value to `null` since the prop has to be supplied for the layer to function.
- Add `compare: false` to avoid rerender on shallow change (happens in React if the prop is defined inline).

Reply:
Sounds good